doc/dev/GETTING_STARTED.md

```diff
@@ -122,6 +130,14 @@ cmake --build .
 
  * Double click on the `C:\dev\f3d\build\bin_Debug\f3d.exe` executable
 
+### Running the test suite
+
+```
+cmake -DBUILD_TESTING=ON .
+cmake --build .
```

Comment:
Visual studio is a multi-config generator, you need to specify the config to use (Release or Debug)
```suggestion
cmake --build . --config Release
```